packages/ethereum/tasks/register.ts

```diff
@@ -75,8 +75,12 @@ async function main(
 
       await (await hoprDummyProxy.ownerBatchAddAccounts(nativeAddresses)).wait()
       await (await hoprNetworkRegistry.ownerRegister(nativeAddresses, peerIds)).wait()
-    } else {
+    } else if (opts.task === 'enable') {
+      await (await hoprNetworkRegistry.enableRegistry()).wait()
+    } else if (opts.task === 'disable') {
       await (await hoprNetworkRegistry.disableRegistry()).wait()
```

Comment:
similar to above: Should we add a check on the current `enabled` value before calling the function? SC will revert if the current state is the same as desired state.

Reply:
fixed